examples/plot.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Create plots for privacy evaluation of "shuffling+flipping"."""
+
+from buffling import signal_to_noise_ratio as snr
+import numpy as np
+import pandas as pd
+import seaborn as sns 
+import matplotlib.pyplot as plt
+
+## global parameters
+NUM_PUBS = 4
+FLIP_PROB = 0.05
+# plt.style.use("ggplot")
+
+## dependent parameters
+NUM_LOGO = 2 ** NUM_PUBS
+TRANSIT_MAT = snr.generate_transition_matrix(NUM_PUBS, FLIP_PROB)
```

Comment:
Do you think it better to do the following?
1. Create flag variables for num_pubs and flip_prob, so that users can control those variables and see different simulation results.
2. Put transit_mat into the main method, which takes FLAGS.num_pubs and FLAGS.flip_prob.

Reply:
good idea!